securedrop/tests/test_journalist.py

```diff
@@ -104,15 +105,36 @@ def test_unauthorized_access_redirects_to_login(self):
         resp = self.client.get(url_for('index'))
         self.assertRedirects(resp, url_for('login'))
 
-    def test_invalid_credentials(self):
+    def test_login_throttle(self):
```

Comment:
This is analogous to `test_throttle_login` in `test_db.py`, but tests the behavior instead of the implementation (thanks @garrettr for recently reminding me of this important distinction).